java-manta-examples/src/main/java/SimpleClientEncryption.java

```diff
@@ -43,7 +43,7 @@ public static void main(String... args) throws IOException {
                 .setEncryptionPrivateKeyBytes(Base64.getDecoder().decode("RkZGRkZGRkJEOTY3ODNDNkM5MUUyMjIyMTExMTIyMjI="));
 
         try (MantaClient client = new MantaClient(config)) {
-            String mantaPath = "/" + mantaUserName + "/stor/foo";
+            String mantaPath = MantaClient.SEPARATOR + mantaUserName + "/stor/foo";
```

Comment:
We should probably use the separator for the slashes in the /stor/foo string as well.

Reply:
Done.